plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -73,6 +73,7 @@ public class IcebergConfig
     private double minimumAssignedSplitWeight = 0.05;
     private Optional<String> materializedViewsStorageSchema = Optional.empty();
     private boolean sortedWritingEnabled = true;
+    private int maxConcurrentMetadataLoaders = 20;
```

Comment:
Document where this number comes from.

Reply:
I don't have much a of a rationale here besides "that's what number the Hive executor pools default to" https://github.com/trinodb/trino/blob/master/plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java#L88